src/Microsoft.TemplateEngine.Abstractions/Installer/InstallRequest.cs

```diff
@@ -6,6 +6,10 @@ namespace Microsoft.TemplateEngine.Abstractions.Installer
 {
     public class InstallRequest
     {
+        public const string NuGetSourcesKey = "NuGetSources";
+        public const char NuGetSourcesSeparator = ';';
+        public const string InteractiveModeKey = "Interactive";
+
```

Comment:
I'm not fan of `InstallRequest` knowing anything about `NuGet`, hence `Details` being `Dictionary<string,string>`.
But I'm also not sure where to put this, maybe expose it in `NuGetInstallerFactory`